ui/components/src/Legend/Legend.stories.tsx

```diff
@@ -0,0 +1,126 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Legend, LegendProps } from '@perses-dev/components';
+import { action } from '@storybook/addon-actions';
+import { Box } from '@mui/material';
+import { red, orange, yellow, green, blue, indigo, purple } from '@mui/material/colors';
+
```

Comment:
Using a handful of colors from the MUI palette instead of completely random numbers to generate something that looks like a nice rainbow for simple examples.